lib/index.js

```diff
@@ -229,7 +229,7 @@ module.exports = function nuts(opts) {
 
                     // Change filename to use download proxy
                     .map(function(entry) {
-                        entry.filename = getBaseDownloadUrl(req) + entry.version+'/'+entry.filename;
+                        entry.filename = getBaseDownloadUrl(req) + '/' + entry.version+'/'+entry.filename;
```

Comment:
It wasn't.  This is a change I made (to support proxying).  We weren't doing Windows releases before so this never got called.